restapi/src/main/java/io/stargate/web/docsapi/service/ExecutionProfile.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.web.docsapi.service;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.List;
+import org.codehaus.jackson.annotate.JsonProperty;
+import org.codehaus.jackson.annotate.JsonPropertyOrder;
+import org.immutables.value.Value;
+
+@JsonSerialize(as = ImmutableExecutionProfile.class)
+@JsonDeserialize(as = ImmutableExecutionProfile.class)
+@JsonPropertyOrder({"description", "queries", "nested"})
+@Value.Immutable(lazyhash = true)
+public interface ExecutionProfile {
+
+  @JsonProperty
+  String description();
```

Comment:
Is this reterned from the service? Can we also add api model props here, so that swagger is correctly updated?

Reply:
Annotation added